README.md

```diff
@@ -3807,3 +3807,27 @@ My  [Repository for Kottans Front-End](https://github.com/Chotogde/kottans-front
 also known as
 cat .git/config
 is a useful command by the way
+
+
+## Hello, my name is Bogdan and I study Git
```

Comment:
wow, so many Bohdans, do you really need them? )))

Reply:
Sorry, I'll fix it now)